api/src/routes/member.routes.ts

```diff
@@ -13,6 +13,10 @@ memberRoutes.post('/is_banned',
   (request, response) => memberController.isBanned(request, response));
 memberRoutes.get('/getrolename',
   (request, response) => memberController.getPrimaryRoleName(request, response));
+memberRoutes.get('/getAdminLevel',
+  (request, response) => memberController.getAdminLevel(request, response));
+memberRoutes.get('/getdonorlevel',
```

Comment:
Some of these routes are camel cased, some not. We need to keep things consistent.

Reply:
Will fix